apps/docs/src/pages/components/_meta.ts

```diff
@@ -13,6 +13,7 @@ export default {
   input: "Input",
   loading: "Loading",
   loadingContainer: "LoadingContainer",
+  overflowTooltip: "OverflowTooltip",
```

Comment:
no longer necessary (conflict). also, we should call the file `overflow-tooltip.mdx` as [URLs should be kebab-cased](https://redocly.com/docs/cli/rules/paths-kebab-case)

Reply:
@zettca made the changes you mentioned.